src/Console/SshCommand.php

```diff
@@ -49,7 +49,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $host = $this->deployer->hosts->get($input->getArgument('hostname'));
+        $firstHostname = $this->deployer->hosts->getIterator()->current()->getHostname();
```

Comment:
I think it's ok only if there only one host. Otherwise it's implicitly, and explicitly is always better. Inveractive mode maybe solution, but i don't like -i option, maybe it's better to be by default:
* dep ssh - interactive
* dep ssh host